src/Symfony/Component/ExpressionLanguage/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+6.0.0
+-----
+
+* added support of `nullsafe operator` syntax when parsing object's properties
+* added support of `nullsafe operator` syntax when parsing object's methods
```

Comment:
Add instead of added. Let's merge both lines also

Reply:
[Done!](https://github.com/symfony/symfony/pull/45795/commits/bffdcd22cabac5e105697b44e8e34c080dbe5f39)